manualtesting/manual_testing.go

```diff
@@ -22,8 +22,8 @@ import (
 type TestEnvironment struct {
```

Comment:
Would be open on fixing the last issue in `manualtesting` and then remove it from the exception list?

```
manualtesting/manual_testing.go:79:10: if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary) (golint)
```

Reply:
I've just pushed the update.